### ISSUE TITLE
Merge Curated and AI Tables

### ISSUE THREAD

**Issue by phantom-ink** [access-ci-org/ACCESS-SDS#46]
-Bring the best features of the AI-Generated table into the Curated table:
1) Core Features
2) Tags
3) Example Use
-Clearly denote in Column Header which fields are AI-Generated (I recommend the Star emoji used by Google)
-Make sure these changes are reflected in the software description modal (or else it will likely break)

-Once all the information we want from the Dynamic table is merged into Curated, remove it.
